### ISSUE TITLE
How to reallocate partitions after movving to bigger USB flash stick

### ISSUE THREAD

**Issue by Ygarr** [nomadbsd/NomadBSD#66]
At 8Gb live USB stick with NomadBSD I have free space (Showing in file manager)  ~500Mb 
With gnome-disks I have moved successfully  image to 16Gb live USB stick. But still free ~500Mb and ~7 Gb unallocated.
Gparted, partitionmanager(kde) or blivet-gui cannot do anything with UFS partitions.

Is there way to make NomadBSD OS occupy all new memory without full reformatting and reinstalling? Thanks

**Comment by a-bateman** [comment]
I have not tried it but this may be helpful:

`The following properties can be set at creation time and import time, and
     later changed with	the zpool set command:

     autoexpand=on | off
	 Controls automatic pool expansion when	the underlying LUN is grown.
	 If set	to "on", the pool will be resized according to the size	of the
	 expanded device. If the device	is part	of a mirror or raidz then all
	 devices within	that mirror/raidz group	must be	expanded before	the
	 new space is made available to	the pool. The default behavior is
	 "off".	 This property can also	be referred to by its shortened	column
	 name, expand.`

**Comment by bsdlme** [comment]
You can resize the UFS partition with [growfs(8)](http://man.freebsd.org/growfs).

@a-bateman: autoexpand is a good suggestion but it only works with ZFS while the NomadBSD live system is UFS-based.

**Comment by a-bateman** [comment]
You're right.  I've had it installed so long I completely forgot that it was UFS as a live system.  Thanks for the reminder.
